docs/guide/uploading-downloading-binary-content.md

```diff
@@ -0,0 +1,147 @@
+# Uploading & Downloading Binary Content | AngularFire Guide
+
+## Table of Contents
+
+* [Overview](#overview)
+* [API Summary](#api-summary)
+* [Uploading files](#uploading-files)
```

Comment:
All other guide pages use all-caps for the section titles. Please update these to make them consistent.